docs/fundamentals/code-analysis/style-rules/ide0064.md

```diff
@@ -14,17 +14,19 @@ dev_langs:
 ---
 # Make struct fields writable (IDE0064)
 
-|Property|Value|
-|-|-|
-| **Rule ID** | IDE0064 |
-| **Title** | Make struct fields writable |
-| **Category** | CodeQuality |
-| **Subcategory** | Language rules (modifier preferences) |
-| **Applicable languages** | C# |
+| Property                 | Value                                 |
+|--------------------------|---------------------------------------|
+| **Rule ID**              | IDE0064                               |
+| **Title**                | Make struct fields writable           |
+| **Category**             | CodeQuality                           |
+| **Subcategory**          | Language rules (modifier preferences) |
+| **Applicable languages** | C#                                    |
 
 ## Overview
 
-This rule detects structs that contain one ore more `readonly` fields and also contains an assignment to `this` outside of the constructor. The rule recommends converting `readonly` fields to non-readonly, i.e. writable. Marking such struct fields as readonly can lead to unexpected behavior, as the value assigned to the field can change when `this` is assigned outside the constructor.
+This rule detects structs that contain one or more `readonly` fields and also contain an assignment to `this` outside of the constructor. The rule recommends converting `readonly` fields to non-readonly, that is, writable. Marking such struct fields as `readonly` can lead to unexpected behavior, because the value assigned to the field can change when `this` is assigned outside the constructor.
```

Comment:
I think the example should be more clear about what "unexpected behavior" could happen (ie, illustrate how the two structs can behave in different ways)

Reply:
Feel free to submit a new PR to show an example, but since it's not directly related to this issue and would require some research on my part, I'm going to merge this as is.

Reply:
I have tried to find some example but couldn't. Probably @sharwell can suggest an example?